README.md

```diff
@@ -1,5 +1,5 @@
 # hw01
-# **Story of my life**
+# **Story of My Life**
 
 I am Wei, a psychology graduate student trying to learn how to use R. I am from Changsha, a southern city in China that always rains in the spring and summer time. I do environment-related research here in the University of Chicago. [Here is a link to my lab website ](http://enl.uchicago.edu/) where you can learn more about my projects.
 
```

Comment:
Looks like you included five different markdown elements (bold, header, link embed, image embed, and a list), and they all rendered perfectly!

Reply:
Great job with the spacing on all of these elements, since if the spacing is off even by a little, things could get wonky. For example, the bullets might not have tabbed in the right amount if the spacing hadn't been done correctly. Also, you clearly put the star trek meme in the right repo and knew to commit it along with the README file, which is something that took me a little while to figure out.